apps/i2c-accelerometer-gyroscope/src/i2c-accelerometer-gyroscope.c

```diff
@@ -0,0 +1,161 @@
+/* i2c-accelerometer-gyroscope.c - KNoT Application Client */
+
+/*
+ * Copyright (c) 2020, CESAR. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * This example uses I2C communication protocol to get acceleration and angular
+ * velocity from a gy-251 sensor.
+ *
+ */
+
+#include <zephyr.h>
+#include <net/net_core.h>
+#include <logging/log.h>
+#include <device.h>
+#include <i2c.h>
+
+#include "knot.h"
+#include <knot/knot_types.h>
+#include <knot/knot_protocol.h>
+
+/* I2C slave address */
+#define MPU_ADDRESS			0X68
+
+/* MPU6050 internal registers */
+#define MPU6050_REG_PWR_MGMT1		0x6B
+#define MPU6050_SLEEP_EN		BIT(6)
+#define MPU6050_REG_GYRO_CFG		0x1B
+#define MPU6050_REG_ACCEL_CFG		0x1C
+#define MPU6050_REG_DATA_START		0x3B
```

Comment:
Align the tabulation between macro and value
```suggestion
#define MPU6050_REG_DATA_START	0x3B
```